components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationMetadataService.java

```diff
@@ -151,6 +165,27 @@ public OIDCMetaData getOIDCMetadata() {
         OIDCMetaData oidcMetaData = new OIDCMetaData();
         OAuthAdminServiceImpl oAuthAdminService = ApplicationManagementServiceHolder.getOAuthAdminService();
 
+        List<String> tokenEpAuthMethods = OAuth2Util.getSupportedClientAuthenticationMethods();
```

Comment:
Need have  metadata on the available subject types and the default subject type

Reply:
Fixed in 1f4de9bb1b4a721922a631b048ee16397e81baea